src/app/use-root-startup.ts

```
@@ -0,0 +1,166 @@
+    let refreshIsRTL: (() => void) | undefined;
+    let i18nInitSucceeded = false;
+    try {
+      const i18nModule = await import('@/lib/i18n');
+      await i18nModule.initI18n();
+      i18nInitSucceeded = true;
+      applyRTLIfNeeded = i18nModule.applyRTLIfNeeded;
+      refreshIsRTL = i18nModule.refreshIsRTL;
+    } catch {
+      // non-fatal
+    } finally {
+      if (!isMounted) return;
```

Biome (2.1.2)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)